instrumentation/cassandra/cassandra-3.0/javaagent/src/test/java/CassandraClientTest.java

```diff
@@ -57,10 +61,12 @@ static void beforeAll() {
             .withStartupTimeout(Duration.ofMinutes(2));
     cassandra.start();
 
+    cassandraHost = cassandra.getHost();
+    cassandraIp = java.net.InetAddress.getByName(cassandra.getHost()).getHostAddress();
```

Comment:
is there need for fully-qualifying `java.net.InetAddress`?

Reply:
I'm not sure, sometimes my IntelliJ would randomly do that instead of adding an import. I assumed it had a reason, but I can go back and fix them if necessary.

Reply:
we generally avoid fully-qualifying if not needed, if you don't mind removing where it's not needed it would be appreciated

Reply:
Done